src/Battleship.java

```diff
@@ -48,12 +48,25 @@ public static void main(String[] args) {
             int number = currentMovePlayer1 ? 1 : 2;
             String[][] mapToCheck = currentMovePlayer1 ? player2 : player1;
 
-            System.out.println();
-            System.out.println("Введите координаты для атаки игрок_" + number  + "- x,y");
-            String line = sc.nextLine(); //x,y;
+            int x = 0;
+            int y = 0;
```

Comment:
It's no need to declare there variables before cycle. You should declare it before while on 53 line. But it's critical error